src/tags/everyone.js

```diff
@@ -0,0 +1,30 @@
+/*
+ * @Author: stupid cat
+ * @Date: 2017-05-07 18:38:55
+ * @Last Modified by: stupid cat
+ * @Last Modified time: 2017-10-19 18:12:29
+ *
+ * This project uses the AGPLv3 license. Please read the license file before using/adapting any of the code.
+ */
+
+const Builder = require('../structures/TagBuilder');
+
+module.exports =
+    Builder.BotTag('everyone')
```

Comment:
You can possibly think about adding an alias to this subtag. Calling the subtag `everyonemention` and adding the alias `everyone` would make more sense in my opinion.